src/Magnum/DebugTools/TextureImage.cpp

```diff
@@ -56,7 +57,7 @@ static void importDebugToolsResources() {
 namespace Magnum { namespace DebugTools {
 
 #if defined(MAGNUM_TARGET_GLES) && !defined(MAGNUM_TARGET_GLES2)
-using namespace Containers::Literals;
+using namespace Corrade::Containers::Literals;
```

Comment:
This part shouldn't be needed I think, right? Only the include.

Reply:
Hmm yep it seems that you are right